hardware/scm/g80.md

```diff
@@ -1,95 +1,182 @@
-
-| Baud | Propagation | Phase1 | Phase2 | Baudrate Prescaler | Synchronization Jump Width | Use Multi Bit Sampling | Sample Point | Max Osc. Tolerance | Max Cable Length
-|---|---|---|---|---|---|---|---|---|---
-| 33.333K | 1 | 12 | 2 | 90 | 1 | False | 86.7%  | 0.33% | 2145M
-| 83.333K | 1 | 12 | 2 | 36 | 1 | False | 86.7%  | 0.33% | 825M
-| 125K    | 1 | 12 | 2 | 24 | 1 | False | 86.7%  | 0.33% | 531M
-| 250K    | 1 | 12 | 2 | 12 | 1 | False | 86.7%  | 0.33% | 238M
-| 500K    | 1 | 12 | 2 | 6  | 1 | False | 86.7%  | 0.33% | 91M
-| 1M      | 1 | 12 | 2 | 3  | 1 | False | 86.7%  | 0.33% | 18M
-
-*Note: Maximum Oscillator Tolerance and Maximum Cable Length are theoretical maximums and must be tested to ensure reliability.*
-
-## G80 Development Board
-The G80 Dev Board is an excellent starting point and reference design for anyone interested in evaluating and developing with the G80.
-
-![G80 Dev Board](images/g80dev.jpg)
-
-* [Schematics](http://files.ghielectronics.com/downloads/Schematics/Systems/G80%20Dev%20Board%20Schematic.pdf)
-
-## Legal
-### License
-This System on Module with its TinyCLR OS firmware and libraries are licensed for commercial and noncommercial use. No additional fee or licensing is required. None of the provided documentation/schematics/libraries/sources are to be used in any manner outside for the benefit of using and implementing this System on Module into your product. Please see the [FEZ](../fez/intro.md) for an open source TinyCLR OS reference design. The [TinyCLR Licensing](../../software/tinyclr/licensing.md) page has more details.
-
-### Disclaimer
-
-IN NO EVENT SHALL GHI ELECTRONICS, LLC BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS PRODUCT, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE. GHI ELECTRONICS, LLC LINE OF PRODUCTS ARE NOT DESIGNED FOR LIFE SUPPORT APPLICATIONS. SPECIFICATIONS, PRICE AND AVAILABILITY ARE SUBJECT TO CHANGE WITHOUT ANY NOTICE.
-
-***
-
-TinyCLR you program our devices (and others) in C# or Visual Basic using Microsoft's Visual Studio -- and it's all free!  [**Learn more...**](../../software/tinyclr/intro.md).
-
+﻿# G80
+---
+
+![G80](images/g80-noborder.jpg)
+
+## Overview
+
+The G80 SoC is a low-cost System on Chip part of the GHI Electronics SurfaceMount Compute Modules (SCM) line. This modules can be programmed in .NET C# and Visual Basic using TinyCLR OS or .NET Micro Framework.

```

Comment:
Pick one of MicroFramework vs Micro Framework. We use both in this PR. I think Micro Framework is the correct branding.